conf/base/catalog.yml

```
@@ -30,6 +30,10 @@ oa.data_collection.downstream.cites.intermediate:
   <<: *_js_ptd
   path: "s3://alphafold-impact/data/02_intermediate/oa/works/downstream/${globals:oa.data_collection.get.work_id}/"
 
+oa.data_collection.works_for_concepts_and_years.raw:
+  <<: *_pq
+  filepath: s3://alphafold-impact/data/01_raw/openalex/works/oa_works_for_concepts_and_years.parquet
```

I thought we had agreed to label things `oa`, not `openalex`. If so:
```suggestion
  filepath: s3://alphafold-impact/data/01_raw/oa/works/oa_works_for_concepts_and_years.parquet
```